src/test/java/bio/terra/service/snapshotbuilder/utils/CriteriaQueryBuilderTest.java

```diff
@@ -169,7 +169,7 @@ void generateFilterForCriteriaGroupHandlesMustMeetFalse() {
     SnapshotBuilderCriteriaGroup criteriaGroup =
         new SnapshotBuilderCriteriaGroup()
             .criteria(List.of(generateListCriteria(), generateRangeCriteria()))
-            .meetAll(false)
+            .meetAll(true)
```

Comment:
This change is why it didn't flag before as an issue. Because the test was `true` `true`, and `false` `false` the incorrect flag check didn't cause any sql problems - I changed it.